source/Patches/Roles/Role.cs

```diff
@@ -547,6 +586,20 @@ private static void UpdateMeeting(MeetingHud __instance)
                     }
                     else
                     {
+                        //Remove red color for impostors in meeting if anonymous impostor
+                        var playerChar = PlayerControl.AllPlayerControls.ToArray()
+                        .FirstOrDefault(x => x.PlayerId == player.TargetPlayerId);
+                        var localPlayer = PlayerControl.LocalPlayer;
+                        if (!localPlayer.Data.IsDead && localPlayer != player && player != null && playerChar.Data != null)
+                        {
+                            //Change the player name color for darker color to the medic in meeting with the GetColorType Method
+                            if (localPlayer.Is(RoleEnum.Medic) &&
+                                GetColorType(playerChar) == "darker" &&
```

Comment:
The final project is to display the suspect colors only if the report displays the color tint,
to display "guilty" or something of the kind if the medic has the name, and of course nothing if he has no info, except the timing which could be found, towards the place where the votes of the mayor are displayed

Reply:
to replace completely the chat report or in complement

Reply:
Ideally it should display the suspect colors only if the medic has *ever* received a report with colors this game. That actually shouldn't be that hard; it can track in the Medic class.

Reply:
yes, but it was second time that I wanted to do that.
first the aim was to make it easier for the medic to distinguish between darker and lighter colors during meetings

Reply:
I understand not wanting to make that change (although it should be very easy), but in the meantime this needs to be gated behind medic reports on. Otherwise you're changing the display with something that can't help and might confuse players.

Reply:
otherwise as players I have actually noticed that new players wondered why they saw names in black, I had thought to add "darker" or "lighter" under the names, so that it is clearer. but I don't know why I couldn't do it again, yet I did when I was still looking for how to change the color of the nickname.

Reply:
The Other Roles adds (D) or (L) to the names.

Honestly, none of these work well, which is part of my problem with the color matters thing.